qualtran/resource_counting/generalizers.py

```diff
@@ -83,6 +83,9 @@ def ignore_cliffords(b: Bloq) -> Optional[Bloq]:
     from qualtran.bloqs.util_bloqs import ArbitraryClifford
     from qualtran.cirq_interop import CirqGateAsBloq
 
+    if isinstance(b, Adjoint):
+        b = b.subbloq
+
     if isinstance(
         b, (TwoBitSwap, Hadamard, XGate, ZGate, ArbitraryClifford, CNOT, MultiTargetCNOT)
```

Comment:
We can also remove the `isinstance(b, CirqGateAsBloq)` check below by updating the `_cirq_gate_to_bloq` mapping for newly introduced `SGate`

Reply:
done